Resources/translations/SonataCoreBundle.pt_BR.xliff

```diff
@@ -28,575 +28,575 @@
+                <target>Vermelho escuro</target>
             </trans-unit>
             <trans-unit id="darksalmon">
                 <source>darksalmon</source>
-                <target>darksalmon</target>
+                <target>Salmão escuro</target>
             </trans-unit>
             <trans-unit id="darkseagreen">
                 <source>darkseagreen</source>
-                <target>darkseagreen</target>
+                <target>Verde mar escuro</target>
             </trans-unit>
             <trans-unit id="darkslateblue">
                 <source>darkslateblue</source>
-                <target>darkslateblue</target>
+                <target>Azul ardósia escuro</target>
             </trans-unit>
             <trans-unit id="darkslategray">
                 <source>darkslategray</source>
-                <target>darkslategray</target>
+                <target>Cinza ardósia escuro</target>
             </trans-unit>
             <trans-unit id="darkturquoise">
                 <source>darkturquoise</source>
-                <target>darkturquoise</target>
+                <target>Turquesa escura</target>
             </trans-unit>
             <trans-unit id="darkviolet">
                 <source>darkviolet</source>
-                <target>darkviolet</target>
+                <target>Violeta escuro</target>
             </trans-unit>
             <trans-unit id="deeppink">
                 <source>deeppink</source>
-                <target>deeppink</target>
+                <target>Rosa profundo</target>
             </trans-unit>
             <trans-unit id="deepskyblue">
                 <source>deepskyblue</source>
-                <target>deepskyblue</target>
+                <target>Azul céu profundo</target>
             </trans-unit>
             <trans-unit id="dimgray">
                 <source>dimgray</source>
-                <target>dimgray</target>
+                <target>Cinza fosco</target>
             </trans-unit>
             <trans-unit id="dodgerblue">
                 <source>dodgerblue</source>
-                <target>dodgerblue</target>
+                <target>Azul furtivo</target>
             </trans-unit>
             <trans-unit id="firebrick">
                 <source>firebrick</source>
-                <target>firebrick</target>
+                <target>Tijolo refratário</target>
             </trans-unit>
             <trans-unit id="floralwhite">
                 <source>floralwhite</source>
-                <target>floralwhite</target>
+                <target>Branco floral</target>
             </trans-unit>
             <trans-unit id="forestgreen">
                 <source>forestgreen</source>
-                <target>forestgreen</target>
+                <target>Verde floresta</target>
             </trans-unit>
             <trans-unit id="fuchsia">
                 <source>fuchsia</source>
-                <target>fuchsia</target>
+                <target></target>
```

Comment:
Why does this color does not have translation?

Reply:
My mistake, too many files open, missed one translation.